nav2_costmap_2d/test/module_tests.cpp

```diff
@@ -70,7 +70,8 @@ const double RESOLUTION(1);
 const double WINDOW_LENGTH(10);
```

Comment:
Adding some tests to costmap_2d is necessary to show this new feature is working as expected